docs/sources/configuration/_index.md

```diff
@@ -2612,6 +2612,10 @@ shard_streams:
 
 # Minimum number of label matchers a query should contain.
 [minimum_labels_number: <int>]
+
+# The shard size defines how many index gateways should be used for querying.
+# CLI flag: -index-gateway.shard-size
```

Comment:
1) isn't this per-tenant? if so, maybe the CLI flag shouldn't be listed or you can add a clarification saying that the CLI flag usage will only define the default gateway shard size
2) the flag is different from the YAML config. WDYT of renaming it to be consistent with the config?